### ISSUE TITLE
not working in localhost

### ISSUE THREAD

**Issue by shanumithun** [gitscrum-community/laravel-gitscrum#338]
I have recently installed laravel-gitscrum.my env
```
APP_ENV=local
APP_KEY=base64:Y8K7cqYOOxdk4ws+dSME+ilPPs2592UZxsHHjpfu9Dw=
APP_DEBUG=true
APP_LOG_LEVEL=debug
APP_URL=http://localhost
APP_TITLE=GitScrum

GITHUB_CLIENT_ID=key
GITHUB_CLIENT_SECRET=key

GITLAB_KEY=key
GITLAB_SECRET=key
GITLAB_INSTANCE_URI=https://gitlab.com/

DB_CONNECTION=mysql
DB_HOST=localhost
DB_PORT=3306
DB_DATABASE=agile
DB_USERNAME=root
DB_PASSWORD=sds
APP_PAGINATE=10

PROXY_PORT=
PROXY_METHOD=
PROXY_SERVER=
PROXY_USER=
PROXY_PASS=
```
when i hit localhost/projectname/public

i got 
 Welcome to GitScrum

 

 Login with GitHub and gitlab

I am totaly confused how can i up this project in my localhost.now its redirect to github and Gitlab.please help me.any help would be appreciated

**Comment by AgusRdz** [comment]
Did you set up an application in your Github/Gitlab account?
https://github.com/gitscrum-community/laravel-gitscrum#github

**Comment by shanumithun** [comment]
@AgusRdz yep i did all this and up project.but now login form have two login option only.login through GitHub and gitlab.

**Comment by shanumithun** [comment]
how can i login and register using databse value..?is it possible?

**Comment by AgusRdz** [comment]
Well, currently isn't possible because one purpose of this app is promote the collaborative work through services provided, if you think this could be consider a useful feature feel free to create an issue describing your needs.
